windows/Makefile

```diff
@@ -39,10 +39,12 @@ RUBY_CONFIGURE_ARGS := \
 	--prefix="$(shell cygpath -m ${BUILD_PREFIX})" \
 	--enable-install-static-library \
 	--disable-install-doc \
-	--with-out-ext=openssl,readline,dbm,gdbm,win32ole \
+	--with-out-ext=openssl,readline,dbm,gdbm,win32ole,pty,syslog \
 	--with-static-linked-ext \
 	--disable-rubygems \
-	--without-gmp
+	--without-gmp \
+	--with-libyaml-dir="$(shell cygpath -m $(PREFIX))" \
+	--with-libffi-dir="$(shell cygpath -m $(PREFIX))"

```

Comment:
Why are you adding these? And more importantly why only on windows?

Reply:
https://www.ruby-lang.org/en/news/2022/12/25/ruby-3-2-0-released/
Linux don't need it
![image](https://github.com/mkxp-z/mkxp-z/assets/67521199/1dc135e6-b33d-422c-bece-d3dc03868f96)